### PR TITLE
populate level for each item

### PR DESCRIPTION
**Pull request by lamquoc**
Each item is now populated with the current level

### CODE REVIEW

File: src/treebrowser/treebrowser.tpl.html [hunk 1]
```diff
@@ -26,7 +26,7 @@
                 </div>
                 <div ng-show="item._ad_expanded">
                     <div class="tree-level tree-sub-level"
-                         onLoad="level=level+1"
+                         onLoad="level=level+1;item.level=level"
```
Comment (kashjs) [inline src/treebrowser/treebrowser.tpl.html]: _ad_level

File: src/treebrowser/treebrowser.tpl.html [hunk 2]
```diff
@@ -45,7 +45,7 @@
                     </div>
                 </div>
                 <div class="tree-level tree-top-level border"
-                     onload="level = 1"
+                     onload="level = 1;item.level=level"
```
Comment (kashjs) [inline src/treebrowser/treebrowser.tpl.html]: _ad_level

File: src/treebrowser/docs/treeNode.html [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <div>
     <span class="name-cell pull-left">{{ item.name }}</span>
-    <span class="another-cell pull-left">Another Cell</span>
+    <span class="another-cell pull-left">Another Cell at level {{item.level}}</span>
```
Comment (kashjs) [inline src/treebrowser/docs/treeNode.html]: _ad_level

**Comment by lamquoc** [comment]
Done. Thanks,
Lam

**Comment by kashjs** [comment]
hey lam do we still need this?

**Pull request closed without merging**